control/plugin_manager.go

```diff
@@ -355,10 +355,45 @@ func (p *pluginManager) LoadPlugin(details *pluginDetails, emitter gomit.Emitter
 	lPlugin.ConfigPolicy = cp
 
 	if resp.Type == plugin.CollectorPluginType {
+		cfgNode := p.pluginConfig.getPluginConfigDataNode(core.PluginType(resp.Type), resp.Meta.Name, resp.Meta.Version)
+
+		if lPlugin.ConfigPolicy != nil {
+			// Get config policy from root/plugin defaults
+			ncd := lPlugin.ConfigPolicy.Get([]string{""})
```

Comment:
Node config data? Throwing things out there... @geauxvirtual or @danielscottt would know according to git blame.